packages/storage/src/types.ts

```diff
@@ -63,7 +63,7 @@ export type AsyncStorageWithOptions<O> = {
   clear?: () => Promise<void> | void;
   getItem: (key: string, options?: O) => Promise<string | null> | string | null;
   getAll?: () => Promise<any>;
-  setItem: (key: string, value: string, options?: O) => Promise<void> | void;
+  setItem: ((key: string, value: string, options?: O) => Promise<void> | void);
   removeItem: (key: string) => Promise<void> | void;
   key: (index: number) => Promise<string | null> | string | null;
   readonly length: Promise<number> | number | undefined;
```

Comment:
LocalForage's length property is a [_function_ returning a promise](https://github.com/localForage/localForage/blob/master/typings/localforage.d.ts#L26), not a property.

 (unless there's some adapter logic I'm missing.)

Reply:
We're not using length at all in makePersisted. Also, we use localforage's type directly if available, so there's no issue here.